packages/dom/src/dom.js

```diff
@@ -60,14 +60,17 @@ function isSelectionForward( selection ) {
 }
 
 /**
- * Check whether the selection is horizontally at the edge of the container.
+ * Check whether the selection is at the edge of the container. Checks for
+ * horizontal position by default. Set `isVertical` to true to check only
+ * vertically.
  *
- * @param {Element} container Focusable element.
- * @param {boolean} isReverse Set to true to check left, false for right.
+ * @param {Element} container  Focusable element.
+ * @param {boolean} isReverse  Set to true to check left, false to check right.
```

Comment:
This should [be](http://usejsdoc.org/tags-param.html#parameters-with-properties):

```
@param {Element} container ...
@param {Object} opts ...
@param {Object} opts.isReverse ...
@param {Object} opts.isVertical ...
```

Reply:
Fixed in ccce7e0.